### PR TITLE
[JENKINS-40950] Fix CreateItemTest#duplicate_item_name_displays_error test

### PR DESCRIPTION
**Pull request by varyvol**
[JENKINS-40950](https://issues.jenkins-ci.org/browse/JENKINS-40950)

@reviewbybees

### CODE REVIEW

File: src/test/java/core/CreateItemTest.java [hunk 1]
```diff
@@ -5,38 +5,48 @@
...
 import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertThat;
 
 public class CreateItemTest extends AbstractJUnitTest {
+
+    private static final String JOB_NAME = "asdf";
+    private static final String NAME_FIELD = "name";
+    private static final String JOB_CREATION_ERROR_MSG = "A job already exists";
+
+    private static final By NAME_NOT_EMPTY_MSG = By.id("itemname-required");
```
Comment (raul-arabaolaza) [inline src/test/java/core/CreateItemTest.java]: Can  not use the `by` instead of the selenium `By`?? To not mix usses of `by` and `By`

Comment (varyvol) [inline src/test/java/core/CreateItemTest.java]: Sure, I have changed all of them to by.

**Comment by raul-arabaolaza** [comment]
:bee:

**Comment by mafraba** [comment]
🐝

**Pull request merged**